ops/model.py

```diff
@@ -3269,7 +3269,7 @@ def exec(
         stdin: bytes | BinaryIO | None = None,
         stdout: BinaryIO | None = None,
         stderr: BinaryIO | None = None,
-        encoding: None = None,
+        encoding: None,
```

Comment:
We were doing it wrong before.

2 overloads must be distinguishable, while we had default value (different ones) in two overloads, and that was the only different field.

Reply:
> One version has a string that defaults to utf8 and one has None that defaults to None.

Then, which of the two overloads should be picked when no value is specified?